src/View/View.php

```diff
@@ -115,6 +115,13 @@ class View implements EventDispatcherInterface
      */
     protected $helpers = [];
 
+    /**
+     * The nname of the subfolders cotaining all files for this View.
+     *
+     * @var string
```

Comment:
|null
Typo in "nname".

Reply:
I didn't add |null because all the other config variables don't use it. I think that's something to change separately.